.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+---
+name: 'Stale issues and PRs handler'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6.0.1
```

Comment:
can this be
```suggestion
      - uses: actions/stale@v6
```

it'll reduce dependabot noise and for actions is usually recomended to ping to the major version only

Reply:
Updated. Is there a way to test it before merging the PR?

Reply:
> Is there a way to test it before merging the PR?

not that I'm aware of, this is a cron job so not sure how those work on GH actions

Reply:
Just checked their usage https://github.com/marketplace/actions/close-stale-issues#usage, and `v6` is allowed